### PR TITLE
Fix prop type warnings

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA: none the test should pass.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5535/
Test PASSed.

File: jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js [hunk 1]
```diff
@@ -191,9 +193,11 @@ describe('Charmbrowser', function() {
         acl={acl}
         addNotification={addNotification}
         apiUrl={apiUrl}
+        apiVersion="v5"
         appState={appState}
         changeState={changeState}
         charmstoreSearch={sinon.stub()}
+        charmstoreURL="http://1.2.3.4/"
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js]: Oops thanks.

**Comment by anthonydillon** [comment]
LGTM @huwshimi thanks! :+1:

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1821

**Pull request merged**